### PR TITLE
Week 14 Proposal

### PR DESCRIPTION
**Pull request by frivas**
Hi, this is the proposed publication for Week 14. 

Thank you very much :D

### CODE REVIEW

File: week14-18-05-03.md [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+[@jseibert](https://github.com/jseibert)
+
+[vapor/core#118](https://github.com/vapor/core/pull/118)
+
+### Fix infinite recursion in MySQLCharacterSet.description
+[@jseibert](https://github.com/jseibert)
+
+[vapor/mysql#155](https://github.com/vapor/mysql/pull/155)
+
+### Fixed issue with EvenLoop assert
```
Comment (twof) [inline week14-18-05-03.md]: "EvenLoop" -> "EventLoop"

File: week14-18-05-03.md [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+[vapor/console#63](https://github.com/vapor/console/pull/63)
+
+## NIO
+### Optimize for HTTP/1.1 and HTTP/1.0 responses
+Motivation:
+
+Its very likely that the response will be HTTP/1.1 or HTTP/1.0 so we can optimize for it by minimizing the buffer.write(...) calls.
+Beside this we should also change the pattern of *.write(into: inout ByteBuffer) to extension on ByteBuffer itself.
+
+Modificiations:
```
Comment (twof) [inline week14-18-05-03.md]: "Modificiations" -> "Modifications"

File: week14-18-05-03.md [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+
+[vapor/mysql#146](https://github.com/vapor/mysql/pull/146)
+
+### Fix MySQL index syntax
+[@jseibert](https://github.com/jseibert)
+
+[vapor/fluent-mysql#88](https://github.com/vapor/fluent-mysql/pull/88)
+
+
+## Tagged releases
```
Comment (twof) [inline week14-18-05-03.md]: We don't need to do it in this release if you don't have the time, but in future releases, we should link each of these to their tagging on GitHub

**Comment by frivas** [comment]
All the suggested changes were included. Thank you very much

**Pull request merged**